### DISCUSSION TITLE
Engine Sounds at Climb Detent

### DISCUSSION THREAD

**Discussion by pdeswardt** [flybywiresim/aircraft#6267] [Q&A]
Hi, when thrust levers are moved from TOGA to FLEX and then CLIMB ( During takeoff ), the movement from FLEX to CLIMB makes the engines **sound** like they go to **idle** - this does not sound realistic.

I have seen some videos where this is done on other A320 Neo's, and the engines still **sound** like they are going strong, albeit slightly lower in rpm, but not as if you moved the throttles to idle...

I'm running the DEV version ( latest )

Will this be fixed in a future release ?

Thanks in advance for all you guys do !

**Comment by CptBayer** [comment]
Hi,

also whe you are in cruise on a flightlevel and change this to a higher one, the engines spools up to fast when starting the climb.... might be in connection with a not smooth enough rise in vertival speed ... mostly is that the case when climbing from FL100 to a higher one and the aircraft accelerates from 250 to 290 kts ... in higher flightlevel everything is smooth.

greets

**Comment by speedwaystar** [comment]
the massive and alarming drop in volume from TOGA/FLEX to LVR CLMB is a relatively recent change to the A32NX sound pack and not a good one.

**Reply by tracernz** [reply]
> the massive and alarming drop in volume from TOGA/FLEX to LVR CLMB is a relatively recent change to the A32NX sound pack and not a good one.

Nothing changed in the sounds... it's a side-effect of a systems change that made thrust limits much more accurate. There is still more work to do.
